types/jsreport-core/tslint.json

```diff
@@ -1 +1,9 @@
-{ "extends": "dtslint/dt.json" }
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
+		"no-const-enum": false,
```

Comment:
Please remove the const enums, as we are no longer accepting them for most cases.